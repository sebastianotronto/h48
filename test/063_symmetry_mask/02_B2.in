UF0 DB0 UB0 DF0 UR0 UL0 DL0 DR0 FR0 FL0 BR0 BL0 UFR0 DBR0 DFL0 UBL0 UFL0 DBL0 DFR0 UBR0
