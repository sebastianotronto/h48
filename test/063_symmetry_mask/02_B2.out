100010000100010000000000100010000100010000000000
